include/deal.II/fe/fe.h

```diff
@@ -2398,15 +2398,20 @@ class FiniteElement : public Subscriptor,
    *
    * For each component number <tt>c</tt>, the entries have the following
    * meaning: <dl> <dt><tt>table[c].first.first</tt></dt> <dd>Number of the
-   * base element for <tt>c</tt>.</dd> <dt><tt>table[c].first.second</tt></dt>
-   * <dd>Component in the base element for <tt>c</tt>.</dd>
-   * <dt><tt>table[c].second</tt></dt> <dd>Multiple of the base element for
-   * <tt>c</tt>.</dd> </dl>
+   * base element for <tt>c</tt>. This is the index you can pass to
+   * base_element().</dd> <dt><tt>table[c].first.second</tt></dt>
+   * <dd>Component within the base element for <tt>c</tt>. This value is
+   * between 0 and the n_components() of this base element.</dd>
+   * <dt><tt>table[c].second</tt></dt> <dd>Index of the multiple of the base
```

Comment:
-> multiplicity?

Reply:
Oh, no, I see -- it's the *index* of the base element within the multiplicity of this base.

Reply:
yeah, multiple is a strange word, but correct here, I think.

Reply:
Or maybe "the how-manyth copy of the base element" or similar?

In the end it's not that important -- you can glean the meaning by the next sentence where you say that it must be between zero and the multiplicity of the base, and that's what makes it clear.

Reply:
Yes, that is why I added it. I had to look up how it is filled. ;-)